### DISCUSSION TITLE
Problem when editing image data

### DISCUSSION THREAD

**Discussion by Zaksnowman** [sqlitebrowser/sqlitebrowser#3634] [Q&A]
I have writen a python program to convert between different map tile databases.  I have been using DB Browser to inspect the files and remove certain entries.  That works fine, but after I have used it to edit a PNG image of copy one and paste it to another cell, my python program then throws up the error - Exception has occurred: OperationalError Could not decode to UTF-8 column 'image' with text '?PNG.  Any idea what the problem may be.  I have tried a different database editor which didn't cause the problem.  Prefer to use DB Browser as it is a much nicer program to use.

**Comment by mgrojo** [comment]
This might be due to  #3307. Please, try updating to the last release and tell us if it worked.
